README.md

```diff
@@ -54,3 +54,7 @@ Once you compiled or packaged everything you needed, you can stop the container
  The site for each module can be found at <project dir>\target\site\index.html
 
   ![Site](site.png)
+  
+  ## Agreement on Compatibility from now on
+  
+  Interfaces started to be used in this refactoring process, and GeneXus SA commits evolving some of them maintaining compatibility.
```

Comment:
```suggestion
  With interfaces now in place since the v16 Upgrade 4 release, GeneXus SA commits to evolve them while maintaining compatibility.
```
Me parece que pondría algo así para que no quede out-of-date pronto y sea más directo con el compromiso de compatibilidad. Creo que la parte del refactoring es medio irrelevante para este punto.